waybar/config

```diff
@@ -4,8 +4,8 @@
     "height": 30, // Waybar height (to be removed for auto height)
      "width": 1920, // Waybar width
     // Choose the order of the modules
-    "modules-left": ["sway/workspaces", "sway/mode", "custom/media"],
-    "modules-center": ["sway/window"],
+    "modules-left": ["custom/workspaces"],
```

Comment:
Just for my curiosity: what was "custom/media"?

Reply:
No idea, it was on the default config